<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title> Selected publications | Wouter Saelens </title> <meta name="author" content="Wouter Saelens"> <meta name="description" content="A simple, whitespace theme for academics. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="wouter saelens, systems immunology, causal systems immunology, perturb-seq"> <link rel="stylesheet" href="/assets/css/bootstrap.min.css?a4b3f509e79c54a512b890d73235ef04"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="/assets/css/academicons.min.css?f0b7046b84e425c55f3463ac249818f5"> <link defer rel="stylesheet" href="/assets/css/scholar-icons.css?62b2ac103a88034e6882a5be5f3e2772"> <link defer rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons&amp;display=swap"> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-github.css?591dab5a4e56573bf4ef7fd332894c99" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="data:image/svg+xml,&lt;svg%20xmlns=%22http://www.w3.org/2000/svg%22%20viewBox=%220%200%20100%20100%22&gt;&lt;text%20y=%22.9em%22%20font-size=%2290%22&gt;%F0%9F%A7%A9&lt;/text&gt;&lt;/svg&gt;"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://zouter.github.io/publications/"> <script src="/assets/js/theme.js?a81d82887dd692e91686b43de4542f18"></script> <link defer rel="stylesheet" href="/assets/css/jekyll-pygments-themes-native.css?5847e5ed4a4568527aa6cfab446049ca" media="none" id="highlight_theme_dark"> <script>
    initTheme();
  </script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top" role="navigation"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"> <span class="font-weight-bold">Wouter</span> Saelens </a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">About </a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">Selected publications <span class="sr-only">(current)</span> </a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">CV </a> </li> <li class="nav-item "> <a class="nav-link" href="/people/">People </a> </li> <li class="nav-item"> <button id="search-toggle" title="Search" onclick="openSearchModal()"> <span class="nav-link">ctrl k <i class="ti ti-search"></i></span> </button> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="ti ti-sun-moon" id="light-toggle-system"></i> <i class="ti ti-moon-filled" id="light-toggle-dark"></i> <i class="ti ti-sun-filled" id="light-toggle-light"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5" role="main"> <div class="post"> <header class="post-header"> <h1 class="post-title">Selected publications</h1> <p class="post-description"></p> </header> <article> <p>This only lists our main publications. See <a href="https://scholar.google.com/citations?user=msYffYEAAAAJ&amp;hl=nl" rel="external nofollow noopener" target="_blank">Google Scholar</a> for all publications.</p> <script src="/assets/js/bibsearch.js?1bc438ca9037884cc579601c09afd847" type="module"></script> <p><input type="text" id="bibsearch" spellcheck="false" autocomplete="off" class="search bibsearch-form-input" placeholder="Type to filter"></p> <div class="publications"> <h2 class="bibliography">2025</h2> <ol class="bibliography"> <li> <div class="row"> <div class="col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/tfseq.png" class="" width="100%" height="auto" alt="tfseq.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="liuDissectingImpactTranscription2025" class="col-sm-9"> <div class="title"><a href="https://doi.org/10.1038/s41588-025-02343-7" rel="external nofollow noopener" target="_blank">Dissecting the Impact of Transcription Factor Dose on Cell Reprogramming Heterogeneity Using scTF-seq</a></div> <div class="author"> Wangjie Liu<sup>*</sup> , <em>Wouter Saelens<sup>*</sup></em> , Pernille Rainer<sup>*</sup> , Marjan BioÄanin , Vincent Gardeux , Antoni Jakub Gralak , Guido van Mierlo , Angelika Gebhart <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="* contributed equally"> </i> </div> <div class="periodical"> <em>Nature Genetics</em>, Oct 2025 </div> <div class="periodical"> </div> <div class="abstract"> <p>A high-throughput technology to study the effect of transcription factor dose</p> </div> <div class="abstract hidden"> <p>Reprogramming often yields heterogeneous cell fates, yet the underlying mechanisms remain poorly understood. To address this, we developed single-cell transcription factor sequencing (scTF-seq), a single-cell technique that induces barcoded, doxycycline-inducible TF overexpression and quantifies TF dose-dependent transcriptomic changes. Applied to mouse embryonic multipotent stromal cells, scTF-seq generated a gain-of-function atlas for 384 mouse TFs, identifying key regulators of lineage specification, cell cycle control and their interplay. Leveraging single-cell resolution, we uncovered how TF dose shapes reprogramming heterogeneity, revealing both dose-dependent and stochastic cell state transitions. We classified TFs into low-capacity and high-capacity groups, with the latter further subdivided by dose sensitivity. Combinatorial scTF-seq demonstrated that TF interactions can shift from synergistic to antagonistic depending on the relative dose. Altogether, scTF-seq enables the dissection of TF function, dose and cell fate control, providing a high-resolution framework to understand and predict reprogramming outcomes, advancing gene regulation research and the design of cell engineering strategies.</p> </div> </div> <div class="col-sm-8 abbr"> <div class="links"> <a href="https://doi.org/10.1038/s41588-025-02343-7" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“° DOI</a> <a href="https://biblio.ugent.be/publication/01KABRXP674FBEHGA9DT1ADWS9/file/01KABS4VNDH98331NACCRMETTN.pdf" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ’¾ PDF</a> <a href="https://github.com/DeplanckeLab/TF-seq" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘©â€ğŸ’» Code</a> <a href="https://asap.epfl.ch/projects/ASAP96" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸŒ Portal</a> <a href="https://www.ebi.ac.uk/biostudies/arrayexpress/studies/E-MTAB-13010" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“Š Data</a> <a href="www.biorxiv.org/content/10.1101/2024.01.30.577921v1" class="btn z-depth-0" role="button" style="font-size: 1rem;">ğŸ“ Preprint</a> </div> </div> </div> <hr> <script>
  // This script patches the Abstract/Bibtex open buttons.
  // Because the buttons are now in a different column (col-sm-3) than the content (col-sm-7),
  // the default theme script cannot find the content. This helper script bridges that gap.
  $(document).ready(function() {
    // Select buttons that should trigger hidden content
    $(".links a.abstract, .links a.bibtex, .links a.award").click(function() {
        // 1. Find the parent row that contains everything
        var $row = $(this).closest('.row');
        
        // 2. Identify which type of content to open based on the button class
        var type = "abstract";
        if ($(this).hasClass("bibtex")) { type = "bibtex"; }
        if ($(this).hasClass("award")) { type = "award"; }
        
        // 3. Find the hidden div in the sibling column (main-content-col)
        var $target = $row.find("div." + type + ".hidden");
        
        // 4. Toggle the 'open' class
        if ($target.length > 0) {
            $target.toggleClass('open');
            
            // 5. Trigger Masonry layout update if the theme uses it
            if(typeof $grid !== 'undefined') {
                $grid.masonry('layout');
            }
        }
    });
  });
</script> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/funkyheatmap.png" class="" width="100%" height="auto" alt="funkyheatmap.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="cannoodt*FunkyheatmapVisualisingData2025" class="col-sm-9"> <div class="title"><a href="https://doi.org/10.21105/joss.07698" rel="external nofollow noopener" target="_blank">Funkyheatmap: Visualising Data Frames with Mixed Data Types</a></div> <div class="author"> Robrecht Cannoodt<sup>*</sup> , Louise Deconinck<sup>*</sup> , Artuur Couckuyt<sup>*</sup> , Nikolay S. Markov<sup>*</sup> , Luke Zappia , Malte D. Luecken , Marta Interlandi , Yvan Saeys<sup>â€ </sup> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="â€  co-supervised the work"> </i> </div> <div class="periodical"> <em>Journal of Open Source Software</em>, Apr 2025 </div> <div class="periodical"> </div> <div class="abstract"> <p>A clean and easy way to make complex heatmaps in Python, R and Javascript</p> </div> <div class="abstract hidden"> <p>Cannoodt* et al., (2025). funkyheatmap: Visualising data frames with mixed data types. Journal of Open Source Software, 10(108), 7698, https://doi.org/10.21105/joss.07698</p> </div> </div> <div class="col-sm-8 abbr"> <div class="links"> <a href="https://doi.org/10.21105/joss.07698" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“° DOI</a> <a href="https://github.com/funkyheatmap/funkyheatmap" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘©â€ğŸ’» Code</a> <a href="https://pypi.org/project/funkyheatmappy/" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘¨â€ğŸ”¬ Tool</a> </div> </div> </div> <hr> <script>
  // This script patches the Abstract/Bibtex open buttons.
  // Because the buttons are now in a different column (col-sm-3) than the content (col-sm-7),
  // the default theme script cannot find the content. This helper script bridges that gap.
  $(document).ready(function() {
    // Select buttons that should trigger hidden content
    $(".links a.abstract, .links a.bibtex, .links a.award").click(function() {
        // 1. Find the parent row that contains everything
        var $row = $(this).closest('.row');
        
        // 2. Identify which type of content to open based on the button class
        var type = "abstract";
        if ($(this).hasClass("bibtex")) { type = "bibtex"; }
        if ($(this).hasClass("award")) { type = "award"; }
        
        // 3. Find the hidden div in the sibling column (main-content-col)
        var $target = $row.find("div." + type + ".hidden");
        
        // 4. Toggle the 'open' class
        if ($target.length > 0) {
            $target.toggleClass('open');
            
            // 5. Trigger Masonry layout update if the theme uses it
            if(typeof $grid !== 'undefined') {
                $grid.masonry('layout');
            }
        }
    });
  });
</script> </li> <li> <div class="row"> <div class="col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/chromatinhd.png" class="" width="100%" height="auto" alt="chromatinhd.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="saelensChromatinHDConnectsSinglecell2025a" class="col-sm-9"> <div class="title"><a href="https://doi.org/10.1038/s41467-024-55447-9" rel="external nofollow noopener" target="_blank">ChromatinHD Connects Single-Cell DNA Accessibility and Conformation to Gene Expression through Scale-Adaptive Machine Learning</a></div> <div class="author"> <em>Wouter Saelens</em> , Olga Pushkarev , Bart Deplancke </div> <div class="periodical"> <em>Nature Communications</em>, Jan 2025 </div> <div class="periodical"> </div> <div class="abstract"> <p>A scale-adaptive machine learning method to link single-cell chromatin accessibility to gene expression</p> </div> <div class="abstract hidden"> <p>Gene regulation is inherently multiscale, but scale-adaptive machine learning methods that fully exploit this property in single-nucleus accessibility data are still lacking. Here, we develop ChromatinHD, a pair of scale-adaptive models that uses the raw accessibility data, without peak-calling or windows, to link regions to gene expression and determine differentially accessible chromatin. We show how ChromatinHD consistently outperforms existing peak and window-based approaches and find that this is due to a large number of uniquely captured, functional accessibility changes within and outside of putative cis-regulatory regions. Furthermore, ChromatinHD can delineate collaborating regulatory regions, including their preferential genomic conformations, that drive gene expression. Finally, our models also use changes in ATAC-seq fragment lengths to identify dense binding of transcription factors, a feature not captured by footprinting methods. Altogether, ChromatinHD, available at https://chromatinhd.org, is a suite of computational tools that enables a data-driven understanding of chromatin accessibility at various scales and how it relates to gene expression.</p> </div> </div> <div class="col-sm-8 abbr"> <div class="links"> <a href="https://doi.org/10.1038/s41467-024-55447-9" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“° DOI</a> <a href="https://github.com/DeplanckeLab/ChromatinHD_manuscript" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘©â€ğŸ’» Code</a> <a href="https://chromatinhd.org" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘¨â€ğŸ”¬ Tool</a> </div> </div> </div> <hr> <script>
  // This script patches the Abstract/Bibtex open buttons.
  // Because the buttons are now in a different column (col-sm-3) than the content (col-sm-7),
  // the default theme script cannot find the content. This helper script bridges that gap.
  $(document).ready(function() {
    // Select buttons that should trigger hidden content
    $(".links a.abstract, .links a.bibtex, .links a.award").click(function() {
        // 1. Find the parent row that contains everything
        var $row = $(this).closest('.row');
        
        // 2. Identify which type of content to open based on the button class
        var type = "abstract";
        if ($(this).hasClass("bibtex")) { type = "bibtex"; }
        if ($(this).hasClass("award")) { type = "award"; }
        
        // 3. Find the hidden div in the sibling column (main-content-col)
        var $target = $row.find("div." + type + ".hidden");
        
        // 4. Toggle the 'open' class
        if ($target.length > 0) {
            $target.toggleClass('open');
            
            // 5. Trigger Masonry layout update if the theme uses it
            if(typeof $grid !== 'undefined') {
                $grid.masonry('layout');
            }
        }
    });
  });
</script> </li> </ol> <h2 class="bibliography">2022</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/liveratlas.png" class="" width="100%" height="auto" alt="liveratlas.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="guilliamsSpatialProteogenomicsReveals2022" class="col-sm-9"> <div class="title"><a href="https://doi.org/10.1016/j.cell.2021.12.018" rel="external nofollow noopener" target="_blank">Spatial Proteogenomics Reveals Distinct and Evolutionarily Conserved Hepatic Macrophage Niches</a></div> <div class="author"> Martin Guilliams , Johnny Bonnardel , Birthe Haest , Bart Vanderborght , Camille Wagner , Anneleen Remmerie , Anna Bujko , Liesbet Martens , <span class="more-authors" title="click to view 26 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '26 more authors' ? 'Tinne ThonÃ©, Robin Browaeys, Federico F. De Ponti, Bavo Vanneste, Christian Zwicker, Freya R. Svedberg, Tineke Vanhalewyn, Amanda GonÃ§alves, Saskia Lippens, Bert Devriendt, Eric Cox, Giuliano Ferrero, Valerie Wittamer, Andy Willaert, Suzanne J. F. Kaptein, Johan Neyts, Kai Dallmeier, Peter Geldhof, Stijn Casaert, Bart Deplancke, Peter Dijke, Anne Hoorens, Aude Vanlander, Frederik Berrevoet, Yves Van Nieuwenhove, Yvan Saeys' : '26 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.html(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '10'); ">26 more authors</span>, <em>Wouter Saelens<sup>â€ </sup></em>, Hans Van Vlierberghe<sup>â€ </sup>, Lindsey Devisscher<sup>â€ </sup>, and Charlotte L. Scott<sup>â€ </sup> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="* co-supervised the work"> </i> </div> <div class="periodical"> <em>Cell</em>, Jan 2022 </div> <div class="periodical"> </div> <div class="abstract"> <p>One of the first comprehensive liver cell atlases combining single-cell and spatial transcriptomics with proteomics</p> </div> </div> <div class="col-sm-8 abbr"> <div class="links"> <a href="https://doi.org/10.1016/j.cell.2021.12.018" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“° DOI</a> <a href="https://biblio.ugent.be/publication/8738275/file/8738278.pdf" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ’¾ PDF</a> <a href="https://www.livercellatlas.org/" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸŒ Portal</a> </div> </div> </div> <hr> <script>
  // This script patches the Abstract/Bibtex open buttons.
  // Because the buttons are now in a different column (col-sm-3) than the content (col-sm-7),
  // the default theme script cannot find the content. This helper script bridges that gap.
  $(document).ready(function() {
    // Select buttons that should trigger hidden content
    $(".links a.abstract, .links a.bibtex, .links a.award").click(function() {
        // 1. Find the parent row that contains everything
        var $row = $(this).closest('.row');
        
        // 2. Identify which type of content to open based on the button class
        var type = "abstract";
        if ($(this).hasClass("bibtex")) { type = "bibtex"; }
        if ($(this).hasClass("award")) { type = "award"; }
        
        // 3. Find the hidden div in the sibling column (main-content-col)
        var $target = $row.find("div." + type + ".hidden");
        
        // 4. Toggle the 'open' class
        if ($target.length > 0) {
            $target.toggleClass('open');
            
            // 5. Trigger Masonry layout update if the theme uses it
            if(typeof $grid !== 'undefined') {
                $grid.masonry('layout');
            }
        }
    });
  });
</script> </li></ol> <h2 class="bibliography">2021</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/dyngen.png" class="" width="100%" height="auto" alt="dyngen.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="cannoodtSpearheadingFutureOmics2021" class="col-sm-9"> <div class="title"><a href="https://doi.org/10.1038/s41467-021-24152-2" rel="external nofollow noopener" target="_blank">Spearheading Future Omics Analyses Using Dyngen, a Multi-Modal Simulator of Single Cells</a></div> <div class="author"> Robrecht Cannoodt<sup>*</sup> , <em>Wouter Saelens<sup>*</sup></em> , Louise Deconinck , Yvan Saeys <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="* contributed equally"> </i> </div> <div class="periodical"> <em>Nature Communications</em>, Jun 2021 </div> <div class="periodical"> </div> <div class="abstract"> <p>A flexible simulator for single-cell multi-omics data, useful for benchmarking computational methods</p> </div> <div class="abstract hidden"> <p>We present dyngen, a multi-modal simulation engine for studying dynamic cellular processes at single-cell resolution. dyngen is more flexible than current single-cell simulation engines, and allows better method development and benchmarking, thereby stimulating development and testing of computational methods. We demonstrate its potential for spearheading computational methods on three applications: aligning cell developmental trajectories, cell-specific regulatory network inference and estimation of RNA velocity.</p> </div> </div> <div class="col-sm-8 abbr"> <div class="links"> <a href="https://doi.org/10.1038/s41467-021-24152-2" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“° DOI</a> <a href="https://biblio.ugent.be/publication/8714817/file/8714819.pdf" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ’¾ PDF</a> <a href="https://github.com/dynverse/dyngen" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘©â€ğŸ’» Code</a> <a href="https://github.com/dynverse/dyngen" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘¨â€ğŸ”¬ Tool</a> </div> </div> </div> <hr> <script>
  // This script patches the Abstract/Bibtex open buttons.
  // Because the buttons are now in a different column (col-sm-3) than the content (col-sm-7),
  // the default theme script cannot find the content. This helper script bridges that gap.
  $(document).ready(function() {
    // Select buttons that should trigger hidden content
    $(".links a.abstract, .links a.bibtex, .links a.award").click(function() {
        // 1. Find the parent row that contains everything
        var $row = $(this).closest('.row');
        
        // 2. Identify which type of content to open based on the button class
        var type = "abstract";
        if ($(this).hasClass("bibtex")) { type = "bibtex"; }
        if ($(this).hasClass("award")) { type = "award"; }
        
        // 3. Find the hidden div in the sibling column (main-content-col)
        var $target = $row.find("div." + type + ".hidden");
        
        // 4. Toggle the 'open' class
        if ($target.length > 0) {
            $target.toggleClass('open');
            
            // 5. Trigger Masonry layout update if the theme uses it
            if(typeof $grid !== 'undefined') {
                $grid.masonry('layout');
            }
        }
    });
  });
</script> </li></ol> <h2 class="bibliography">2020</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/nichenet.png" class="" width="100%" height="auto" alt="nichenet.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="browaeysNicheNetModelingIntercellular2020" class="col-sm-9"> <div class="title"><a href="https://doi.org/10.1038/s41592-019-0667-5" rel="external nofollow noopener" target="_blank">NicheNet: Modeling Intercellular Communication by Linking Ligands to Target Genes</a></div> <div class="author"> Robin Browaeys , <em>Wouter Saelens<sup>â€ </sup></em> , Yvan Saeys<sup>â€ </sup> <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="â€  co-supervised the work"> </i> </div> <div class="periodical"> <em>Nature Methods</em>, Feb 2020 </div> <div class="periodical"> </div> <div class="abstract"> <p>A widely used method to predict cell-cell communication from single-cell data by not only looking at ligand-receptor pairs but also modeling downstream target gene regulation</p> </div> <div class="abstract hidden"> <p>Computational methods that model how gene expression of a cell is influenced by interacting cells are lacking. We present NicheNet (https://github.com/saeyslab/nichenetr), a method that predicts ligandâ€“target links between interacting cells by combining their expression data with prior knowledge on signaling and gene regulatory networks. We applied NicheNet to tumor and immune cell microenvironment data and demonstrate that NicheNet can infer active ligands and their gene regulatory effects on interacting cells.</p> </div> </div> <div class="col-sm-8 abbr"> <div class="links"> <a href="https://doi.org/10.1038/s41592-019-0667-5" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“° DOI</a> <a href="https://drive.google.com/file/u/0/d/1YoMkqJ4ftaHXJjz83yzosx72XYgpfMVG/view" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ’¾ PDF</a> <a href="https://github.com/saeyslab/nichenetr" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘©â€ğŸ’» Code</a> <a href="https://github.com/saeyslab/nichenetr" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘¨â€ğŸ”¬ Tool</a> </div> </div> </div> <hr> <script>
  // This script patches the Abstract/Bibtex open buttons.
  // Because the buttons are now in a different column (col-sm-3) than the content (col-sm-7),
  // the default theme script cannot find the content. This helper script bridges that gap.
  $(document).ready(function() {
    // Select buttons that should trigger hidden content
    $(".links a.abstract, .links a.bibtex, .links a.award").click(function() {
        // 1. Find the parent row that contains everything
        var $row = $(this).closest('.row');
        
        // 2. Identify which type of content to open based on the button class
        var type = "abstract";
        if ($(this).hasClass("bibtex")) { type = "bibtex"; }
        if ($(this).hasClass("award")) { type = "award"; }
        
        // 3. Find the hidden div in the sibling column (main-content-col)
        var $target = $row.find("div." + type + ".hidden");
        
        // 4. Toggle the 'open' class
        if ($target.length > 0) {
            $target.toggleClass('open');
            
            // 5. Trigger Masonry layout update if the theme uses it
            if(typeof $grid !== 'undefined') {
                $grid.masonry('layout');
            }
        }
    });
  });
</script> </li></ol> <h2 class="bibliography">2019</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/dynbenchmark.png" class="" width="100%" height="auto" alt="dynbenchmark.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="saelensComparisonSinglecellTrajectory2019" class="col-sm-9"> <div class="title"><a href="https://doi.org/10.1038/s41587-019-0071-9" rel="external nofollow noopener" target="_blank">A Comparison of Single-Cell Trajectory Inference Methods</a></div> <div class="author"> <em>Wouter Saelens<sup>*</sup></em> , Robrecht Cannoodt<sup>*</sup> , Helena Todorov , Yvan Saeys <i class="fa-solid fa-circle-info ml-1" data-toggle="popover" data-placement="top" data-html="true" data-content="* contributed equally"> </i> </div> <div class="periodical"> <em>Nature Biotechnology</em>, May 2019 </div> <div class="periodical"> </div> <div class="abstract"> <p>The reference benchmark paper for single-cell trajectory inference methods</p> </div> <div class="abstract hidden"> <p>Trajectory inference approaches analyze genome-wide omics data from thousands of single cells and computationally infer the order of these cells along developmental trajectories. Although more than 70 trajectory inference tools have already been developed, it is challenging to compare their performance because the input they require and output models they produce vary substantially. Here, we benchmark 45 of these methods on 110 real and 229 synthetic datasets for cellular ordering, topology, scalability and usability. Our results highlight the complementarity of existing tools, and that the choice of method should depend mostly on the dataset dimensions and trajectory topology. Based on these results, we develop a set of guidelines to help users select the best method for their dataset. Our freely available data and evaluation pipeline (https://benchmark.dynverse.org) will aid in the development of improved tools designed to analyze increasingly large and complex single-cell datasets.</p> </div> </div> <div class="col-sm-8 abbr"> <div class="links"> <a href="https://doi.org/10.1038/s41587-019-0071-9" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“° DOI</a> <a href="https://drive.google.com/file/u/0/d/10MXw4uzI7a-aCfp8GlWCgi97ocfork4Y/view" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ’¾ PDF</a> <a href="https://github.com/dynverse/dynbenchmark" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘©â€ğŸ’» Code</a> <a href="https://github.com/dynverse/dyno" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘¨â€ğŸ”¬ Tool</a> <a href="https://www.biorxiv.org/content/10.1101/276907v1" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“ Preprint</a> </div> </div> </div> <hr> <script>
  // This script patches the Abstract/Bibtex open buttons.
  // Because the buttons are now in a different column (col-sm-3) than the content (col-sm-7),
  // the default theme script cannot find the content. This helper script bridges that gap.
  $(document).ready(function() {
    // Select buttons that should trigger hidden content
    $(".links a.abstract, .links a.bibtex, .links a.award").click(function() {
        // 1. Find the parent row that contains everything
        var $row = $(this).closest('.row');
        
        // 2. Identify which type of content to open based on the button class
        var type = "abstract";
        if ($(this).hasClass("bibtex")) { type = "bibtex"; }
        if ($(this).hasClass("award")) { type = "award"; }
        
        // 3. Find the hidden div in the sibling column (main-content-col)
        var $target = $row.find("div." + type + ".hidden");
        
        // 4. Toggle the 'open' class
        if ($target.length > 0) {
            $target.toggleClass('open');
            
            // 5. Trigger Masonry layout update if the theme uses it
            if(typeof $grid !== 'undefined') {
                $grid.masonry('layout');
            }
        }
    });
  });
</script> </li></ol> <h2 class="bibliography">2018</h2> <ol class="bibliography"><li> <div class="row"> <div class="col-sm-2 abbr"> <figure> <picture> <img src="/assets/img/publication_preview/module.png" class="" width="100%" height="auto" alt="module.png" data-zoomable="" loading="eager" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> </div> <div id="saelensComprehensiveEvaluationModule2018" class="col-sm-9"> <div class="title"><a href="https://doi.org/10.1038/s41467-018-03424-4" rel="external nofollow noopener" target="_blank">A Comprehensive Evaluation of Module Detection Methods for Gene Expression Data</a></div> <div class="author"> <em>Wouter Saelens</em> , Robrecht Cannoodt , Yvan Saeys </div> <div class="periodical"> <em>Nature Communications</em>, Mar 2018 </div> <div class="periodical"> </div> <div class="abstract"> <p>A benchmark of gene module detection methods</p> </div> <div class="abstract hidden"> <p>A critical step in the analysis of large genome-wide gene expression datasets is the use of module detection methods to group genes into co-expression modules. Because of limitations of classical clustering methods, numerous alternative module detection methods have been proposed, which improve upon clustering by handling co-expression in only a subset of samples, modelling the regulatory network, and/or allowing overlap between modules. In this study we use known regulatory networks to do a comprehensive and robust evaluation of these different methods. Overall, decomposition methods outperform all other strategies, while we do not find a clear advantage of biclustering and network inference-based approaches on large gene expression datasets. Using our evaluation workflow, we also investigate several practical aspects of module detection, such as parameter estimation and the use of alternative similarity measures, and conclude with recommendations for the further development of these methods.</p> </div> </div> <div class="col-sm-8 abbr"> <div class="links"> <a href="https://doi.org/10.1038/s41467-018-03424-4" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ“° DOI</a> <a href="https://www.nature.com/articles/s41467-018-03424-4.pdf" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ’¾ PDF</a> <a href="https://github.com/saeyslab/moduledetection-evaluation" class="btn z-depth-0" role="button" style="font-size: 1rem;" rel="external nofollow noopener" target="_blank">ğŸ‘©â€ğŸ’» Code</a> </div> </div> </div> <hr> <script>
  // This script patches the Abstract/Bibtex open buttons.
  // Because the buttons are now in a different column (col-sm-3) than the content (col-sm-7),
  // the default theme script cannot find the content. This helper script bridges that gap.
  $(document).ready(function() {
    // Select buttons that should trigger hidden content
    $(".links a.abstract, .links a.bibtex, .links a.award").click(function() {
        // 1. Find the parent row that contains everything
        var $row = $(this).closest('.row');
        
        // 2. Identify which type of content to open based on the button class
        var type = "abstract";
        if ($(this).hasClass("bibtex")) { type = "bibtex"; }
        if ($(this).hasClass("award")) { type = "award"; }
        
        // 3. Find the hidden div in the sibling column (main-content-col)
        var $target = $row.find("div." + type + ".hidden");
        
        // 4. Toggle the 'open' class
        if ($target.length > 0) {
            $target.toggleClass('open');
            
            // 5. Trigger Masonry layout update if the theme uses it
            if(typeof $grid !== 'undefined') {
                $grid.masonry('layout');
            }
        }
    });
  });
</script> </li></ol> </div> </article> </div> </div> <footer class="fixed-bottom" role="contentinfo"> <div class="container mt-0"> Â© Copyright 2026 Wouter Saelens. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="/assets/js/bootstrap.bundle.min.js"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@5.0.0/imagesloaded.pkgd.min.js" integrity="sha256-htrLFfZJ6v5udOG+3kNLINIKh2gvoKqwEhHYfTTMICc=" crossorigin="anonymous"></script> <script defer src="/assets/js/masonry.js?a0db7e5d5c70cc3252b3138b0c91dcaf" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.1.0/dist/medium-zoom.min.js" integrity="sha256-ZgMyDAIYDYGxbcpJcfUnYwNevG/xi9OHKaR/8GK+jWc=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js?85ddb88934d28b74e78031fd54cf8308"></script> <script src="/assets/js/no_defer.js?2781658a0a2b13ed609542042a859126"></script> <script defer src="/assets/js/common.js?c15de51d4bb57887caa2c21988d97279"></script> <script defer src="/assets/js/copy_code.js?c8a01c11a92744d44b093fc3bda915df" type="text/javascript"></script> <script defer src="/assets/js/jupyter_new_tab.js?d9f17b6adc2311cbabd747f4538bb15f"></script> <script async src="https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js"></script> <script async src="https://badge.dimensions.ai/badge.js"></script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.2/es5/tex-mml-chtml.js" integrity="sha256-MASABpB4tYktI2Oitl4t+78w/lyA+D7b/s9GEP0JOGI=" crossorigin="anonymous"></script> <script src="/assets/js/mathjax-setup.js?a5bb4e6a542c546dd929b24b8b236dfd"></script> <script defer src="https://cdnjs.cloudflare.com/polyfill/v3/polyfill.min.js?features=es6" crossorigin="anonymous"></script> <script defer src="/assets/js/progress-bar.js?2f30e0e6801ea8f5036fa66e1ab0a71a" type="text/javascript"></script> <script src="/assets/js/vanilla-back-to-top.min.js?f40d453793ff4f64e238e420181a1d17"></script> <script>
    addBackToTop();
  </script> <script type="module" src="/assets/js/search/ninja-keys.min.js?a3446f084dcaecc5f75aa1757d087dcf"></script> <ninja-keys hidebreadcrumbs noautoloadmdicons placeholder="Type to start searching"></ninja-keys> <script src="/assets/js/search-setup.js?6c304f7b1992d4b60f7a07956e52f04a"></script> <script src="/assets/js/search-data.js"></script> <script src="/assets/js/shortcut-key.js?6f508d74becd347268a7f822bca7309d"></script> </body> </html>